.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+⚠️ If you do not respect this template, your pull request will be closed.
+⚠️ Your pull request title should be short detailed and understandable for all.
+⚠️ Also, please add a release note file using reno if the change needs to be
+  documented in the release notes.
+⚠️ If your pull request fixes an open issue, please link to the issue.
+
+- [ ] I have added the tests to cover my changes.
+- [ ] I have updated the documentation accordingly.
+- [ ] I have read the CONTRIBUTING document.
+-->
+
+### Summary
+
```

Comment:
What about adding here "Fix #<number-of-the-issue>"? It would help not to forget to add it